### ISSUE TITLE
Database’s 

### ISSUE THREAD

**Issue by roi35** [RandstormBTC/randstorm#10]
is only one address used? Is it possible to insert a database of addresses for searching?

**Comment by turalyf** [comment]
You can use it with a fork, it’s the same, just with a database.

[](https://github.com/lukmansamosir89/randstorm)

**Comment by RandstormBTC** [comment]
Yes I made it so it generates the seed 10 minutes before the Epoch. I am still experimenting with the correct random number generation. It seems I may have to use Javascript rather than implementing the SecureRandom function in Python. 

![seeds](https://github.com/RandstormBTC/randstorm/assets/153292960/ceccddbc-e6b7-450c-8874-494faae8d034)

**Comment by demonluca** [comment]
> Yes I made it so it generates the seed 10 minutes before the timestamp of the first transaction. I am still experimenting with the correct random number generation. It seems I may have to use Javascript rather than implementing the SecureRandom function in Python.

Is the program not creating the correct seeds we need, because JavaScript isn’t been used?

**Comment by turalyf** [comment]
> Yes I made it so it generates the seed 10 minutes before the timestamp of the first transaction. I am still experimenting with the correct random number generation. It seems I may have to use Javascript rather than implementing the SecureRandom function in Python. 
> 
> 
> 
> ![seeds](https://github.com/RandstormBTC/randstorm/assets/153292960/ceccddbc-e6b7-450c-8874-494faae8d034)
> 
> 

please fix the **"target_address"** error when running randstorm.py

**Comment by RandstormBTC** [comment]
> Is the program not creating the correct seeds we need, because JavaScript isn’t been used?

It looks like that is a good possibility. I am running tests with all the old projects that I can find from 2010 - 2014 time period that use javascript to generate an address.

**Comment by roi35** [comment]
if I want to search all addresses, from 2010 to 2016, what is the correct period, please help me indicate, is this value incorrect? with these dates?start_timestamp=[1262304000](tel:1262304000), end_timestamp=[1388534399](tel:1388534399))

**Comment by roi35** [comment]
We would like to test it there. when should we expect the release?

**Comment by randomchoice2** [comment]
> > Is the program not creating the correct seeds we need, because JavaScript isn’t been used?
> 
> It looks like that is a good possibility. I am running tests with all the old projects that I can find from 2010 - 2014 time period that use javascript to generate an address.

Unfortunately I also think so. We have to write the old Math.random() algo which is called mwc1616, I think it will be also faster & it uses timestamp for the first time as seed as well.

What am I not understanding yet is how to decrease entropy bits, in general how to write a code that "reverses" from input to rng creation and priv key generation. It requires much more deeper understanding of cryptography

**Comment by roi35** [comment]
Will there be a ready-made version for use? with loading the address database..? because it was early. thank you in advance for doing the work, we will thank you tips)

**Comment by roi35** [comment]
Can you do it the way it was before, load the address database and have the script sort through the range randomly? click on Generate and write down private keys in a notebook?

**Comment by roi35** [comment]
Can someone explain how to use it? nothing is clear... the author is silent

**Comment by fcinumber81** [comment]
Has anyone came up with a resolution to this script? Please let me know, thanks

**Comment by mikejnpc** [comment]
ya i would like to know also... doesnt  seem to generate correctly
